### PR TITLE
WINUI-2816 _Ribbon comboBox UG document

### PR DESCRIPTION
**Pull request by SaranRajChandrasekaran**
Added the RibbonComboBox control in the WinUI UG document.

### CODE REVIEW

File: winui/Ribbon/Getting-Started.md [hunk 1]
```diff
@@ -382,6 +382,58 @@ this.rootGrid.Children.Add(sfRibbon);
 
 N> The other icon types such as   [BitmapIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.bitmapicon?view=windows-app-sdk-1.0), [FontIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.fonticon?view=windows-app-sdk-1.0), [PathIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.pathicon?view=windows-app-sdk-1.0) and [SymbolIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.symbolicon?view=windows-app-sdk-1.0) can be used for the Ribbon items such as RibbonButton, RibbonDropDownButton, RibbonSplitButton and Ribbon Item host controls.
 
+## Add ribbon ComboBox
+
+The `RibbonComboBox` control combines a text box and a drop-down list box, allowing users to choose items from the list or add new values and it allows to display in the normal, simplified, and overflow layout of the ribbon. Use the `DisplayOptions` property to set the control in different layouts.
+
```
Comment (jeganraj-m) [inline winui/Ribbon/Getting-Started.md:388]: how control display in overflow menu is missing

File: winui/Ribbon/Getting-Started.md [hunk 2]
```diff
@@ -382,6 +382,58 @@ this.rootGrid.Children.Add(sfRibbon);
 
 N> The other icon types such as   [BitmapIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.bitmapicon?view=windows-app-sdk-1.0), [FontIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.fonticon?view=windows-app-sdk-1.0), [PathIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.pathicon?view=windows-app-sdk-1.0) and [SymbolIcon](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.symbolicon?view=windows-app-sdk-1.0) can be used for the Ribbon items such as RibbonButton, RibbonDropDownButton, RibbonSplitButton and Ribbon Item host controls.
 
+## Add ribbon ComboBox
```
Comment (jeganraj-m) [inline winui/Ribbon/Getting-Started.md:385]: Add ribbon combobox

File: winui/Ribbon/Getting-Started.md [hunk 3]
```diff
@@ -382,6 +382,58 @@ this.rootGrid.Children.Add(sfRibbon);
...
+
+ribbonComboBox1.Items.Add(ribbonComboBoxItemSize);
+ribbonComboBox1.Items.Add(ribbonComboBoxItemSize1);
+ribbonComboBox1.Items.Add(ribbonComboBoxItemSize2);
+rootGrid.Children.Add(ribbonComboBox);
+rootGrid.Children.Add(ribbonComboBox1);
+{% endhighlight %}
+{% endtabs %}
+
+![Add ribbon comboBox in group](Getting-Started-images/ribbonComboBox.png)
```
Comment (jeganraj-m) [inline winui/Ribbon/Getting-Started.md:435]: image doesn't looks good

Comment (SaranRajChandrasekaran) [reply winui/Ribbon/Getting-Started.md]: Image changed

File: winui/Ribbon/Getting-Started.md (hunk 2 above)
Comment (SaranRajChandrasekaran) [reply winui/Ribbon/Getting-Started.md]: Changed casing in title

File: winui/Ribbon/Getting-Started.md (hunk 1 above)
Comment (SaranRajChandrasekaran) [reply winui/Ribbon/Getting-Started.md:388]: Added about the overflow menu UI.

**Review by jeganraj-m — APPROVED** [review]

**Comment by jeganraj-m** [comment]
rebuild

**Pull request merged**